DataManagementSystem/Utilities/DMSHelpers.py

```diff
@@ -155,7 +155,8 @@ def getStorageElements( self ):
 
   def isSEFailover( self, storageElement ):
     seList = resolveSEGroup( self.__opsHelper.getValue( 'DataManagement/SEsUsedForFailover', [] ) )
-    return storageElement in resolveSEGroup( seList )
+    # FIXME: remove string test at some point
+    return storageElement in resolveSEGroup( seList ) or ( not seList and isinstance( storageElement, basestring ) and 'FAILOVER' in storageElement.upper() )
```

Comment:
This means that if we define the SEsUsedForFailover attribute, they all MUST be there, otherwise even if called something FAILOVER, it won't pass the test. Correct ?